homeassistant/components/renson/sensor.py

```diff
@@ -94,39 +93,39 @@ class RensonSensorEntityDescription(
     ),
     RensonSensorEntityDescription(
         key="AIR_FIELD",
-        name="Air quality",
+        translation_key="air_quality",
         field=AIR_QUALITY_FIELD,
         state_class=SensorStateClass.MEASUREMENT,
         raw_format=True,
         native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
     ),
     RensonSensorEntityDescription(
         key="CURRENT_LEVEL_FIELD",
-        name="Ventilation level",
+        translation_key="ventilation_level",
         field=CURRENT_LEVEL_FIELD,
         raw_format=False,
         device_class=SensorDeviceClass.ENUM,
         options=["Off", "Level1", "Level2", "Level3", "Level4", "Breeze", "Holiday"],
```

Comment:
Those as well

Reply:
The downside here is that the names of fields in the translation file must be lowercase, so I can't add these. They also come directly from the API/lib, so I can't really change it myself